observations/obs_converters/AVISO/convert_aviso.f90

```diff
@@ -61,7 +61,7 @@ program convert_aviso
 integer, parameter :: num_copies = 1,   &   ! number of copies in sequence
                       num_qc     = 1        ! number of QC entries
 
-integer  :: ncid, varid, ios
+integer  :: ncid, varid, ios, length
```

Comment:
You don't seem to be using length. 
length is an optional argument to GET_COMMAND_ARGUMENT

Reply:
Length is used in one of the calls to GET_COMMAND_ARGUMENT, but is not really leveraged in the code and can be removed. I'll do this.